thoth/common/logging.py

```diff
@@ -30,7 +30,7 @@
 from typing import Any
 
 from jsonformatter import JsonFormatter
-from sentry_sdk import init as sentry_sdk_init
+from sentry_sdk import init as sentry_sdk_init  # type: ignore[attr-defined]
```

Comment:
Why `ignore` was needed here?

Reply:
When I run `pre-commit` locally, I see `thoth/common/logging.py:33:1: error: Module 'sentry_sdk' has no attribute 'init' [attr-defined]`, but looks like it is not shown when here, https://github.com/thoth-station/common/pull/1007#issuecomment-721309718